analyzer.py

```diff
@@ -114,7 +114,8 @@ def main():
     max_time = max(coding_sessions)
     mean_time = total_time / len(coding_sessions)
     median_time = sorted(coding_sessions.copy())[len(coding_sessions) // 2]
-    total_elapsed = submission_times[-1] - compile_times[0]
+    # it's possible that people submitted before compiling (test cases maybe?)
```

Comment:
This actually isn't possible, per each project's spec: "You must recompile and git push at least once between submissions."